### PR TITLE
Added post talking about docs and the NumPy CZI grant.

### PR DESCRIPTION
**Pull request by melissawm**

### CODE REVIEW

File: posts/2020/03/documentation-as-a-way-to-build-community.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+So, if everybody agrees on its importance, why is it so hard to write good documentation?
+
+## Why do we lack documentation?
+
+In a recent article about documentation for open source data analytics libraries, [Geiger et al.](https://doi.org/10.1007/s10606-018-9333-1) points out that "In a [2017 GitHub survey](https://opensourcesurvey.org/2017/) of OSS contributors, 93% reported that “incomplete or outdated documentation is a pervasive problem” but “60% of contributors say they rarely or never contribute to documentation” (Zlotnick et al. 2017)." At the same time, still in the words of Geiger et al., "Many interviewees who regularly contribute documentation to such projects stated that they did not feel like they received same levels of positive community feedback for documentation work as they did for adding new features or fixing bugs." In fact, the authors in this paper go as far as saying that documentation is the *invisible work* behind these projects.
```
Comment (rgommers) [inline posts/2020/03/documentation-as-a-way-to-build-community.md]: minor: "points out" -> "point out"

Comment (rgommers) [reply posts/2020/03/documentation-as-a-way-to-build-community.md]: The two levels of quotation marks is a little confusing. How about using italics for the inner level to help with that?

File: posts/2020/03/documentation-as-a-way-to-build-community.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Coming from an academic background, the same kind of dynamics seems to apply. Writing a paper or doing an experiment is far more appealing than writing a textbook or developing high-quality educational materials. But educational materials can have a huge impact and effectively bring people into the community. For me, realizing this was a turning point on the way I looked at documentation. 
+
+## Who writes the docs?
+
+If we look at proprietary or company-backed software projects, often there are professional technical writers working on the docs. Having access to these professionals to do the documentation can make a huge difference. However, even then there can be problems. In her excellent talk ["Who Writes the Docs?"](https://www.youtube.com/watch?v=eOC6rsizIvM), Beth Aitman says "People who work on docs often don't feel like their work is valued (...) Being in a position where your value is questioned is pretty horrible."
```
Comment (rgommers) [inline posts/2020/03/documentation-as-a-way-to-build-community.md]: same here and in next paragraph, italics for quote may be nice for readability

File: posts/2020/03/documentation-as-a-way-to-build-community.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+In practical terms, we propose reorganizing the docs into the four categories mentioned in Daniele Procida's article ["What nobody tells you about documentation"](https://www.divio.com/blog/documentation), namely Tutorials, How-Tos, Reference Guide and Explanations. We believe that this will have several consequences:
+- Improving on the quality and discoverability of the documentation as a whole;
+- Showing a clearer difference between documentation aimed at different users (novices vs. experts, for example)
+- Giving users more opportunities to contribute, generating content that can be shared directly on NumPy's official documentation
+- Diversifying our contributor's base, allowing people from different levels of expertise and different life experiences to contribute. This is also extremely important so that we have a better understanding of our community and are able to be accessible, unbiased and welcoming to all people.
```
Comment (rgommers) [inline posts/2020/03/documentation-as-a-way-to-build-community.md]: minor: "contributor's base" -> "contributor base"

File: posts/2020/03/documentation-as-a-way-to-build-community.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
```
Comment (rgommers) [inline posts/2020/03/documentation-as-a-way-to-build-community.md]: There's something you can add here that's in the grant related to the docs-core-team issue you identified earlier: creating a "documentation team" as a first-class team in the project. It's governance deliverable 1 in the proposal: _" Define explicit roles in the project structure and adding a mechanism for devolved decision making on code, documentation, website, and fundraising."_

Comment (melissawm) [reply posts/2020/03/documentation-as-a-way-to-build-community.md]: Thanks!

**Comment by rgommers** [comment]
LGTM now, merged. Thanks Melissa!

**Pull request merged**